pricing.html

```diff
@@ -0,0 +1,267 @@
+
+                </div>
+
+            </div>
+            <!-- card three -->
+
+            <div class="hero-content__card">
+                <div class="hero-content__card-title ">
+                    <img src="img/plan-icon3.svg " alt=" " class="hero-content__icon ">
+                    <div class="hero-content__profile ">
+                        <h1 class="hero-content__card-name ">Regular </h1>
+                        <p class="hero-content__card-surname ">Starter Plan</p>
+
+                    </div>
+                </div>
+                <ul class="hero-content__list  ">
+
+                    <li class="hero-content__item "><img src="img/plan-iconlist.svg " alt=" " class="hero-content__list-icon "> <span>Limited Projects</span></li>
+                    <li class="hero-content__item "><img src="img/plan-iconlist.svg " alt=" " class="hero-content__list-icon "> <span>Regular Support Business</span></li>
+                    <li class="hero-content__item "><img src="img/plan-iconlist.svg " alt=" " class="hero-content__list-icon "> <span>1 month Free Trial</span></li>
+                    <li class="hero-content__item "><img src="img/plan-iconlist.svg " alt=" " class="hero-content__list-icon "> <span>3GB storage</span></li>
+                    <li class="hero-content__item "><img src="img/plan-iconlist.svg " alt=" " class="hero-content__list-icon "> <span>Ads Preview</span></li>
+
+                </ul>
+                <p class="hero-content__card-subtitle ">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</p>
+                <div class="hero-content__pricing ">
+                    <h1 class="hero-content__price ">Free</h1>
+                    <p class="hero-content__card-subtitle ">Billing Monthly</p>
+                    <a href=" " class="hero-content__btn ">Get Started</a>
+
+                </div>
+
+            </div>
+
+        </div>
+    </div>
+    <div class="faq ">
+        <div class="faq__content ">
+
+            <h1 class="faq__title faq__title--large  ">
+                Frequestly Asked Questions
+            </h1>
+            <p class="faq__subtitle ">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt.
+            </p>
+
+        </div>
+        <div class="faq__cards ">
+
+            <div class="faq__active ">
+                <div class="faq__info ">
+                    <h1 class="faq__card-title">
+                        How do I grow my business?
+                    </h1>
+                    <img src="img/Expander.svg " alt=" " class="faq__icon ">
+                </div>
+
+                <p class="faq__subtitle ">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</p>
+            </div>
+            <!-- card two -->
+            <div class="faq-card ">
+                <h1 class="faq__card-title ">
```

Comment:
`faq__card-title` is outside the parent block. here the parent block is  `faq-card`